x-pack/plugins/endpoint/server/routes/alerts/details/handlers.ts

```diff
@@ -22,22 +22,29 @@ export const alertDetailsHandlerWrapper = function(
     try {
       const alertId = req.params.id;
       const response = (await ctx.core.elasticsearch.dataClient.callAsCurrentUser('get', {
+        // Remove the hard coded reference to an alert index here
+        // https://github.com/elastic/endpoint-app-team/issues/311
```

Comment:
would you please remove this comment and instead link to this line of code in the issue.